device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C28S4/BALANCED/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,52 @@
+{%- set default_cable = '300m' %}
+
+{%- macro generate_port_lists(PORT_ALL) %}
+    {# Generate list of ports #}
+    {%- for port_idx in range(0,32) %}
+        {%- if PORT_ALL.append("Ethernet%d" % (port_idx * 4)) %}{%- endif %}
+    {%- endfor %}
+{%- endmacro %}
+
+{%- macro generate_buffer_pool_and_profiles() %}
+    "BUFFER_POOL": {
+        "ingress_lossless_pool": {
+            "size": "32732160",
+            "type": "ingress",
+            "mode": "dynamic",
+            "xoff": "1622016"
+        },
+        "egress_lossy_pool": {
+            "size": "24709632",
+            "type": "egress",
+            "mode": "dynamic"
+        },
+        "egress_lossless_pool": {
+            "size": "32599040",
+            "type": "egress",
+            "mode": "static"
+        }

```

Comment:
In this config, egress_lossless has less buffer compared to ingress lossless. So, there could be egress drop before PFC gets generated.

For now, we could go with buffer configs in device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-D48C8/BALANCED/buffers_defaults_t0.j2

Reply:
Agreed, default should be files under balanced folder.

Reply:
Hi Ying, the default is already in balance folder. But the problem is even that is not working right.

Reply:
We need to switch from C32 to D48C8.

Reply:
Updated now.